### PR TITLE
docs(readme): update FAQ

### PR DESCRIPTION
**Pull request by aslushnikov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -236,40 +236,76 @@ Check out [contributing guide](https://github.com/GoogleChrome/puppeteer/blob/ma
...
 
-We've reached out to a number of these projects to see if there are opportunities for collaboration, and we're happy to do what we can to help.
+This is not an artificial constraint: A lot of work on Puppeteer is actually taking place in the Chromium repository. Here’s a typical story:
+- A Puppeteer bug is reported: https://github.com/GoogleChrome/puppeteer/issues/2709
+- It turned out this is an issue with the DevTools protocol, so we’re fixing it in Chromium: https://chromium-review.googlesource.com/c/chromium/src/+/1102154 
+- Once the upstream fix is landed, we roll updated Chromium into Puppeteer: https://github.com/GoogleChrome/puppeteer/pull/2769
+
+However, oftentimes it is desirable to use Puppeteer with the official Google Chrome rather than Chromium. For this to work, you should pick the version of Puppeteer that uses the Chromium version close enough to Chrome. You can check for Puppeteer's Chromium revision in Release notes or in version selector on [pptr.dev](https://pptr.dev).
+
+#### Q: What’s “Navigation”?
```
Comment (ebidel) [inline README.md]: What's considered a "Navigation"?

File: README.md [hunk 2]
```diff
@@ -236,40 +236,76 @@ Check out [contributing guide](https://github.com/GoogleChrome/puppeteer/blob/ma
...
+
+From Puppeteer’s standpoint, **“navigation” is anything that changes a page’s URL**.
+Aside from regular navigation where the browser hits the network to fetch a new document from the web server, this includes [anchor navigations](https://www.w3.org/TR/html5/single-page.html#scroll-to-fragid) and [History API](https://developer.mozilla.org/en-US/docs/Web/API/History_API) usage.
+
+With this definition of “navigation,” **Puppeteer works seamlessly with single-page applications.**
+
+#### Q: What’s a “trusted input”?
```
Comment (ebidel) [inline README.md]: What’s the difference between a “trusted" and "untrusted" input event?

File: README.md [hunk 3]
```diff
@@ -236,40 +236,76 @@ Check out [contributing guide](https://github.com/GoogleChrome/puppeteer/blob/ma
 
 # FAQ
 
-#### Q: Which Chromium version does Puppeteer use?
-
-Look for `chromium_revision` in [package.json](https://github.com/GoogleChrome/puppeteer/blob/master/package.json).
```
Comment (ebidel) [inline README.md:241]: Think we should keep this one. "Which Chromium version does Puppeteer use?" so people know how to lookup the version.

Comment (aslushnikov) [reply README.md:241]: Brought it back right after "Why puppeteer vXXX doesn't work with chromium vYYY"

**Comment by aslushnikov** [comment]
@ebidel how do you feel about this?

**Review by ebidel — APPROVED** [review]

**Pull request merged**